main.go

```diff
@@ -101,9 +108,31 @@ func updateBenchmarks() {
 	}
 }
 
-func runBenchmarks() (Set, error) {
+func versionRequired(required, tag string) bool {
+	if required == "" {
+		return true
+	}
+	tagVer, _ := semver.Make(trimTagVersion(tag))
+
+	if strings.HasPrefix(required, ">=") {
+		requiredVer, _ := semver.Make(trimTagVersion(strings.TrimLeft(required, ">=")))
+		return tagVer.GTE(requiredVer)
+	}
+	if strings.HasPrefix(required, ">") {
+		requiredVer, _ := semver.Make(trimTagVersion(strings.TrimLeft(required, ">")))
+		return tagVer.GT(requiredVer)
+	}
+	requiredVer, _ := semver.Make(trimTagVersion(required))
+	return tagVer.Equals(requiredVer)
```

Comment:
It would probably be nicer to use a regex here to be more tolerant when it comes to whitespaces and formatting, but this can be done in a follow-up PR

Reply:
Good idea. Added TrimSpace.Will implement it using a regex in a follow-up PR.